service/src/test/java/io/camunda/service/ElementInstanceServiceTest.java

```diff
@@ -188,5 +197,42 @@ public void shouldReturnFlowNodeInstanceWithElementIdAsDefaultName() {
       // then
       assertThat(foundEntity.flowNodeName()).isEqualTo(entity.flowNodeId());
     }
+
+    @Nested
+    class SearchIncidents {
```

Comment:
💭 Adding 'given', 'when', and 'then' comments in the tests helps to see what the "method under test" is. I'd propose to add those comments to the new tests.